spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/users/0/posts'
+      expect(response).to have_http_status(:success)
+      expect(response).to render_template(:index)
+      expect(response.body).to include('Here is list of posts')
+    end
+
+    describe 'GET /show' do
+      it 'returns show page' do
+        get '/users/0/posts/show'
```

Comment:
- [ ] **[OPTIONAL]** I suggest you have a more descriptive path here something like ` get '/users/1/posts/1'`

Reply:
Many thanks for the suggestion. Done!